### ISSUE TITLE
Possible to share S3D model trained on UCF101 -RGB only end to end fine tuned (reported accuracy 81.4)

### ISSUE THREAD

**Issue by Griffintaur** [TengdaHan/CoCLR#35]
Hi Han,
Is it possible to share the UCF-101 RGB model (end to end finetuned) which has been pretrained on UCF-101 and had an accuracy of 81.4?

Thanks,

**Comment by TengdaHan** [comment]
Finetune-CoCLR-ucf101-rgb-128-s3d-ep320.tar
